setup.py

```diff
@@ -25,7 +25,8 @@
 		'argcomplete',
 		'requests',
 		'pyyaml',
-		'jsonschema>=3.2.0'
+		'jsonschema>=3.2.0',
+		'pytest==6.1.2'
```

Comment:
Do we need pytest as a dependency? This should not be necessary for end users, right?

Reply:
Indeed, ``pytest`` is not necessary for end users